src/Chat/Plugin/RFC.php

```diff
@@ -84,6 +84,106 @@ public function search(Command $command): \Generator {
         return $this->chatClient->pinOrUnpinMessage($postedMessage->getMessageId(), $command->getRoom());
     }
 
+    public function getRFC(Command $command): \Generator {
+        $rfc = $command->getParameter(0);
+        if ($rfc === null) {
+            // e.g.: !!rfc pipe-operator
+            return $this->chatClient->postMessage($command->getRoom(), "RFC id required");
+        }
+
+        /*r @var HttpResponse $response */
+        $uri = self::BASE_URI . '/' . urlencode($rfc);
+        $response = yield $this->httpClient->request($uri);
+        if ($response->getStatus() !== 200) {
+            return $this->chatClient->postMessage($command->getRoom(), "Nope, we can't have nice things.");
+        }
+
+        $votes = self::parseVotes($response->getBody());
+        if (empty($votes)) {
+            return $this->chatClient->postMessage($command->getRoom(), "No votes found");
+        }
+
+        $messages = array();
+        foreach ($votes as $id => $vote) {
+            $breakdown = array();
+            $total = array_sum($vote['votes']);
+            if ($total > 0) {
+                foreach ($vote['votes'] as $option => $value) {
+                    $breakdown[] = sprintf("%s (%d: %d%%)", $option, $value, 100 * $value / $total);
+                }
+            }
+            $messages[] = sprintf(
+                "[tag:rfc-vote] [%s](%s) %s",
+                $vote['name'],
+                $uri . '#' . $id,
+                implode(', ', $breakdown)
+            );
+        }
+
+        /** @var PostedMessage $postedMessage */
+        $postedMessage = yield $this->chatClient->postMessage(
+            $command->getRoom(),
+            implode("\n", $messages)
+        );
+
+        $pinnedMessages = yield $this->chatClient->getPinnedMessages($command->getRoom());
```

Comment:
This last bit is just copy-pasta, as I'm not familiar with all the inner workings of the bot.  Hopefully it's not terrible.

![dogscience](https://cloud.githubusercontent.com/assets/812538/15485404/d1d5323e-20f3-11e6-8d47-8bb33ec6b674.jpg)